google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/test/java/com/google/cloud/logging/logback/LoggingAppenderTest.java

```diff
@@ -246,4 +250,28 @@ private LoggingEvent createLoggingEvent(Level level, long timestamp) {
     loggingEvent.setTimeStamp(timestamp);
     return loggingEvent;
   }
+
+  @Test
+  public void testMdcValuesAreConvertedToLabelsWithPassingNullValues() {
+    MDC.put("mdc1", "value1");
+    MDC.put("mdc2", null);
+    MDC.put("mdc3", "value3");
+    logging.setFlushSeverity(Severity.ERROR);
+    Capture<Iterable<LogEntry>> capturedArgument = Capture.newInstance();
+    logging.write(capture(capturedArgument), (WriteOption) anyObject(), (WriteOption) anyObject());
+    expectLastCall().once();
+    replay(logging);
+    Timestamp timestamp = Timestamp.ofTimeSecondsAndNanos(100000, 0);
+    LoggingEvent loggingEvent = createLoggingEvent(Level.INFO, timestamp.getSeconds());
+    loggingAppender.start();
+    loggingAppender.doAppend(loggingEvent);
+    verify(logging);
+    MDC.remove("mdc1");
+    MDC.remove("mdc3");
+    Map<String, String> capturedArgumentMap =
+        capturedArgument.getValue().iterator().next().getLabels();
+    assertEquals("value1", capturedArgumentMap.get("mdc1"));
```

Comment:
Please use `assertThat()` instead of `assertEquals`

Reply:
done